### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by nm17** [jithware/brethap#65] [General Feedback]
This is not an issue, just wanted to say Thank you for this app, continue making it the best meditation app out there and continue adding new features and new fixes. (Also, check out your LiberaPay. Even though this is not much, I still hope this will give you a little bit more motivation for developing this app)

@jithware

**Comment by jithware** [comment]
Thanks @nm17! I'm glad you find the application useful and really appreciate your contribution. Just and FYI. A good place to make comments about the app is in the [discussions](https://github.com/jithware/brethap/discussions) section.
